lstchain/mc/sensitivity.py

```diff
@@ -148,9 +160,18 @@ def calculate_sensitivity_lima(nex, nbg, alpha):
     """
     nex_5sigma = excess_matching_significance_on_off(\
         n_off=nbg,alpha=alpha,significance=5,method='lima')
+
+    for i in range(0, eb):
+        for j in range(0, gb):
+            for k in range(0, tb):
+                if nex_5sigma[i][j][k] < 10:
+                    nex_5sigma[i][j][k] = 10
+                if nex_5sigma[i,j,k] < 0.05 * nbg[i][j][k]/5:
+                    nex_5sigma[i,j,k] = 0.05 * nbg[i][j][k]/5
+
```

Comment:
I do not like this loop inside `calculate_sensitivity_lima`, but I understand why you are putting it here.

The purpose of this function should have been to calculate `nex_5sigma` for a given bg, if this excess does not fulfill certain conditions, it can be changed inside `sens`. The problem is that we would need to recompute the sensitivity there, so it may be ok to leave it here so everything is re-computed and we do not mess so much the main function `sens`